Fabric.Authorization.API/Modules/UsersModule.cs

```diff
@@ -68,6 +79,28 @@ public class UsersModule : FabricModule<User>
             Get("/{identityProvider}/{subjectId}/groups",
                 async _ => await GetUserGroups().ConfigureAwait(false), null,
                 "GetUserGroups");
+
+            Get("/{identityProvider/{subjectId}/roles",
+                async _ => await GetUserRoles().ConfigureAwait(false), null,
+                "GetUserRoles");
+
+            Post("/{identityProvider}/{subjectId}/roles",
+                async param => await AddRolesToUser(param).ConfigureAwait(false), null,
+                "AddRolesToUser");
+        }
+
+        private async Task<dynamic> GetUser(dynamic param)
+        {
+            CheckReadAccess();
+            try
+            {
+                User user = await _userService.GetUser(param.subjectId, param.identityProvider);
+                return user.ToUserApiModel();
+            }
+            catch (NotFoundException<User>)
+            {

```

Comment:
Was an empty string for the message intentional?

Reply:
I'll fix this, not sure why I left it empty.